wam2layers/preprocessing/preprocess_era5_modellevels.py

```diff
@@ -131,4 +133,12 @@ def load_data(variable, date, levels=None):
             "evap": evap,
             "precip": precip,
         }
-    ).to_netcdf(output_path)
+    )
+
+    # Verify that the data meets all the requirements for the model
+    check_input(ds)
+
```

Comment:
You check the input here while you assigned units before, is that not a strange order? I mean ofcourse when you assign the units the unit check will be fine. Or do I misunderstand?

Reply:
You need to assign the units somewhere... xarray often discards them or they get outdated when we do math operations. I want to do the check at the very end, so it acts as a final verification that our preprocessed data meets the requirements for tracking.